README.md

```diff
@@ -1,65 +1,256 @@
+public cloud infrastructure (Hybrid Cloud). The MinIO API maintains compatibility
+with the S3-API to simplify migration of applications onto MinIO Object Storage.
 
-- Kubernetes >= v1.17.0.
-- Create PVs.
-- Install `kubectl minio` plugin using `kubectl krew install minio`.
+## Table of Contents
 
-## Operator Setup
+* [Architecture](https://github.com/minio/operator#Architecture)
+* [Create a MinIO Tenant](https://github.com/minio/operator#Create-a-MinIO-Tenant)
+* [Expand a MinIO Tenant](https://github.com/minio/operator#Expand-a-MinIO-Tenant)
+* [Kubernetes Cluster Configuration](https://github.com/minio/operator#Kubernetes-Cluster-Configuration)
 
-MinIO Operator offers MinIO Tenant creation, management, upgrade, zone addition and more. Operator is meant to control and manage multiple MinIO Tenants.
+# Architecture
 
-To get started, initialize the MinIO Operator deployment. This is a _one time_ process.
+Each MinIO Tenant represents an independent MinIO Object Store service within
+the Kubernetes cluster. The following diagram describes the architecture of as
+MinIO Tenant deployed into Kubernetes:
+
+IMAGE
+
+MinIO provides multiple methods for accessing and managing the MinIO Tenant:
+
+## MinIO Console
+
+The MinIO Console provides a graphical user interface (GUI) for interacting with
+MinIO Tenants. 
+
+IMAGE
+
+MinIO Tenant administrators can perform a variety of tasks through the Console,
+including user creation, policy configuration, and bucket replication. The
+Console also provides a high level view of Tenant health, usage, and healing
+status.
+
+For more complete documentation on using the MinIO Console, see the 
+[MinIO Console Github Repository](https://github.com/minio/console).
+
+## MinIO Operator and `kubectl` Plugin
+
+The MinIO Operator extends the Kubernetes API to support deploying MinIO-specific
+resources as a Tenant in a Kubernetes cluster.
+
+The MinIO `kubectl minio` plugin wraps the Operator to provide a simplified interface
+for deploying and managing MinIO Tenants in a Kubernetes cluster through the
+`kubectl` command line tool.
+
+# Create a MinIO Tenant
+
+This procedure creates a 4-node MinIO Tenant suitable for evaluation and 
+early development using MinIO for object storage.
+
+- MinIO requires Kubernetes version 1.17.0 or later.
+
+- This procedure assumes the cluster contains a 
+  [namespace](https://github.com/minio/operator#create-a-namespace) for
+  the MinIO Tenant.
+
+- This procedure assumes the cluster contains a
+  [`StorageClass`](https://github.com/minio/operator#default-storage-class)
+  for the MinIO Tenant Persistent Volume Claims (`PVC`).
+
+## 1) Install the MinIO Operator
+
+### Install using `kubectl krew`
+
+Run the following command to install the MinIO Operator and Plugin using `krew`:
+
+```sh
+   kubectl krew update
+   kubectl krew install minio
+```
+
+Run the following command to initialize the Operator:
 
 ```sh
 kubectl minio init
+
```

Comment:
This is actually `kubectl minio operator init`, can you fix this as well pls @ravindk89